src/tasks/NetTraceToMibcConverterTask/NetTraceToMibcConverter.csproj

```diff
@@ -0,0 +1,32 @@
+<Project Sdk="Microsoft.NET.Sdk">
```

Comment:
Does this really need to be a separate task?

Can we just add it to `MonoTargetsTasks`? We had a lot of problems in the past due to PackageReference versions.